admin/includes/css/stylesheet.css

```diff
@@ -835,3 +835,49 @@ input.faint {
 .control-label {/*for multiple inputs*/
     font-weight: 700;
 }
+
+.txt-black {
+  color:#000;
+}
+.txt-yellow {
+  color: #ff0;
+}
+.txt-orange {
+  color: #ffa500
+}
+.txt-lime {
+  color:#5ce400
+}
+.txt-red {
+  color:#f00;
+}
+.txt-navy {
+  color: #000080;
+}
+.txt-blue {
+  color: #00f;
+}
+.txt-white {
+  color:#fff;
+}
+.txt-pink {
+  color: #ffbaba;
+}
+.txt-transparent {
+  color: transparent;
+}
+.bold {
```

Comment:
The Bootstrap 4 class for bold-text is `.font-weight-bold`; could we use that instead?